File: app/src/main/java/com/kota/TelnetUI/TextView/TelnetTextViewNormal.java
package com.kota.TelnetUI.TextView;

import android.content.Context;
import android.util.AttributeSet;

public class TelnetTextViewNormal extends TelnetTextView {
    public TelnetTextViewNormal(Context context, AttributeSet attrs, int defStyle) {
        super(context, attrs, defStyle);
        reloadTextSize();
    }

    public TelnetTextViewNormal(Context context, AttributeSet attrs) {
        super(context, attrs);
        reloadTextSize();
    }

    public TelnetTextViewNormal(Context context) {
        super(context);
        reloadTextSize();
    }

    /* access modifiers changed from: protected */
    public void reloadTextSize() {
        setTextModelSize(0);
    }
}

File: app/src/main/java/com/kota/TelnetUI/TextView/TelnetTextViewNormal.kt
package com.kota.TelnetUI.TextView

import android.content.Context
import android.util.AttributeSet

class TelnetTextViewNormal : TelnetTextView {
    constructor(context: Context?, attrs: AttributeSet?, defStyle: Int) : super(
        context,
        attrs,
        defStyle
    ) {
        reloadTextSize()
    }

    constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs) {
        reloadTextSize()
    }

    constructor(context: Context?) : super(context) {
        reloadTextSize()
    }

    /* access modifiers changed from: protected */
    public override fun reloadTextSize() {
        setTextModelSize(TelnetTextView.Companion.TEXT_VIEW_SIZE_NORMAL)
    }
}
